.github/dependabot.yml

```diff
@@ -9,3 +9,13 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+  - package-ecosystem: "cargo"
+    directory: /tools/cp-utility
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+  - package-ecosystem: "docker"
```

Comment:
AFAIK, the `docker` package ecosystem is just for updating image tags in k8s manifests. [docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem).

Reply:
Yes, official docs don't say anything. I was trusting this article: https://dev.to/oracle2025/how-to-keep-a-dockerfile-updated-with-dependabot-1mdn

Reply:
The source code of the dependabot docker package-ecosystem does look for Docker images https://github.com/dependabot/dependabot-core/blob/main/docker/lib/dependabot/docker/file_fetcher.rb